tests/ga/test_update.py

```diff
@@ -2482,6 +2483,17 @@ def __init__(self, return_value=0, side_effect=None):
         self.wait = Mock(return_value=return_value, side_effect=side_effect)
 
 
+class ExtensionsGoalStateMock(object):
```

Comment:
These tests are using deep internals of the UpdateHandler object; I patched them for now, but eventually they should be updated to use the mock_protocol utilities.